Support/Sources/Delivery/Models/SupportContent.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SupportContent.swift
+//  
+//
+//  Created by Martin Dutra on 5/4/22.
+//
+
+import Foundation
+import UIKit
+
+/// An offensive content the user wants to report
+public struct SupportContent {
```

Comment:
I was a little confused when I saw `SupportContent` being references in the code because it made me think it would contain some generic information, basically what `SupportArticle` does. Maybe a more specific name like `OffsensiveContentReport` or something would be more clear. What do you think?

Reply:
I renamed it to OffensiveContent and SupportProfile to AbusiveProfile